source/shared/core_sqlsrv.h

```diff
@@ -172,6 +172,7 @@ const int SQL_SERVER_MAX_FIELD_SIZE = 8000;
 const int SQL_SERVER_MAX_PRECISION = 38;
 const int SQL_SERVER_MAX_TYPE_SIZE = 0;
 const int SQL_SERVER_MAX_PARAMS = 2100;
+const int SQL_MAX_ERROR_MESSAGE_LENGTH = SQL_MAX_MESSAGE_LENGTH * 4;
 
```

Comment:
What's SQL_MAX_MESSAGE_LENGTH? Why multiply it by 4? Perhaps add a simple comment to explain why this new constant is necessary